docs/version-overrides.md

```diff
@@ -1,11 +1,21 @@
 # Version Overrides
 This document lists the version overrides for vulnerable (nested) dependencies and the reason.
 
+## semver
+| Override:           | >=7.5.3 |
```

Comment:
Minor documentation correction. 

The override is still only 7.5.2
But the latest 7.5.3 is consumed.
This is right to consume that latest version because it looks like a vulnerability fix as well, but is not reported as such yet.

Reply:
Thanks @donal-tobin-sap. Corrected with my next commit.